webhooks-extension/config/triggers_prototype/github/webhook.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: webhook-task
+  namespace: tekton-pipelines
+spec:
+  inputs:
+    params:
+    - name: Mode
+      description: Whether or not to create or delete the webhook
+      default: "create"
+    - name: ExternalUrl
+      description: The external access URL (used for creating the webhook)
+      default: "replacemeip.nip.io"
+    - name: GithubOwner 
+      description: The GitHub owner name (used for creating and deleting the webhook)
+      default: "owner"
+    - name: GithubRepo
+      description: The GitHub repository name (used for creating and deleting the webhook)
+      default: "repository"
+    - name: GithubSecretName
+      description: The secret name for GitHub (used for creating and deleting the webhook)
+      default: "githubsecret"
+    - name: GithubAccessTokenKey
+      description: The key name in the secret for the access token (used for creating and deleting the webhook)
+      default: "accessToken"
+    - name: GithubUserNameKey
+      description: The key name in the secret for the user name (used for creating and deleting the webhook)
+      type: string
+    - name: GithubSecretStringKey
+      description: The key name for the GitHub secret string (used for creating and deleting the webhook)
+      default: "secretToken"
+    - name: GithubUrl
+      description: The URL of the GitHub server (used for creating and deleting the webhook)
+      default: "github.com"
+    - name: WebhookName
+      description: The webhook name to delete (as available at (your repository/hooks/<number here>)
+      default: "1234567"
+  steps:  
+  - name: create-or-delete-webhook
+    image: pstauffer/curl:latest   
```

Comment:
have image licenses been checked?

Reply:
Looking at using curlimages/curl:latest as it looks to be official, see https://github.com/curl/curl-docker and note it's under the verified curl org